tensorstore/internal/uri_utils.h

```diff
@@ -16,13 +16,31 @@
 #ifndef TENSORSTORE_INTERNAL_URI_UTILS_H_
 #define TENSORSTORE_INTERNAL_URI_UTILS_H_
 
+#include <cassert>
 #include <cstdint>
 #include <string>
 #include <string_view>
 
+#include "absl/strings/ascii.h"
+
 namespace tensorstore {
 namespace internal {
 
+
+inline int HexDigitToInt(char c) {
```

Comment:
I don't think that this needs to be in the .h?

Reply:
Indeed, placed back in the .cc